### ISSUE TITLE
[Accessibility][Programmatic access] Role of the 'Process selection contacts' should be menu item.

### ISSUE THREAD

**Issue by joschect** [microsoft/fluentui#11012]
Breakout of #6259

**Comment by joschect** [comment]
Only an issue with the controlled picker example.
